SharedSources/ServerBrowsing/UIViewController+VLCAlert.m

```diff
@@ -22,12 +22,14 @@ - (void)vlc_showAlertWithTitle:(NSString *)title message:(NSString *)message but
 #else
 - (void)vlc_showAlertWithTitle:(NSString *)title message:(NSString *)message buttonTitle:(NSString *)buttonTitle
     {
-    VLCAlertView *alert = [[VLCAlertView alloc] initWithTitle:title
-                                                      message:message
-                                                     delegate:self
-                                            cancelButtonTitle:buttonTitle
-                                            otherButtonTitles:nil];
-    [alert show];
+        NSMutableArray<ButtonAction *> *buttonsAction = [[NSMutableArray alloc] init];
```

Comment:
don't use an expensive mutable array here. Creating a static array with a single item is easier and faster.

Reply:
do we really need the extra array here? It might be enough to just use the @[cancelAction] down in the alertViewManagerWithTitle function